drivers/dma/dma_silabs_siwx917.c

```diff
@@ -33,15 +36,16 @@ struct dma_siwx917_config {
 	RSI_UDMA_DESC_T *sram_desc_addr; /* SRAM Address for UDMA Descriptor Storage */
 	const struct device *clock_dev;
 	clock_control_subsys_t clock_subsys;
-	void (*irq_configure)(void);     /* IRQ configure function */
+	void (*irq_configure)(void); /* IRQ configure function */
```

Comment:
This change seems unrelated to this commit. Please keep coding style changes and functional changes separate from each other. This makes it easier to review pull requests.